tests/Fixer/PhpUnit/PhpUnitTestClassRequiresCoversFixerTest.php

```diff
@@ -448,4 +448,62 @@ public static function provideFix82Cases(): iterable
                 PHP,
         ];
     }
+
+    /**
+     * @dataProvider provideFixUnit11Cases
+     *
+     * @requires PHP 8.3
+     */
+    public function testFixUnit11(string $expected, ?string $input = null): void
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public static function provideFixUnit11Cases(): iterable
```

Comment:
Counters in `phpstan.dist.neon` and baseline should only go down, so please add this:

```suggestion
    /**
     * @return iterable<string, array{0: string, 1?: null|string}>
     */
    public static function provideFixUnit11Cases(): iterable
```

and revert previous counter for PHPStan.

Reply:
Will do!